src/modules/esl-modal/test/modal.test.ts

```diff
@@ -0,0 +1,36 @@
+import {ESLModal} from '../core/esl-modal';
+
+describe('ESLModal presence in the body', () => {
+  const $modal = ESLModal.create();
+  $modal.setAttribute('body-inject', 'true');
+  const $container = document.createElement('div');
+  $container.appendChild($modal);
+  const inBody = (): boolean => $modal.parentElement === document.body;
+
+  jest.useFakeTimers();
+  beforeAll(() => {
+    ESLModal.register();
+    document.body.appendChild($container);
+  });
+
+  afterAll(() => {
+    ($modal.parentElement === document.body) && document.body.removeChild($modal);
+  });
+
+  describe('ESLModal with move to body option', () => {
+    test('Call of show function leads to move modal window to body', () => {
+      expect($modal.open).toBe(false);
+      expect(inBody()).toBeFalsy();
+      $modal.show();
+      jest.advanceTimersByTime(1);
+      expect($modal.open).toBe(true);
+      expect(inBody()).toBeTruthy();
+    });
```

Comment:
Too many expectations in the test. It is entirely unclear what exactly is wrong with the code when the test will fail.
Instead of this test, I see the following tests:
 - should open be false on the initial state
 - should InBody() returns false on the initial state
 - should open be true on show state
 - should InBody() returns true on the show state